app/services/atmosphere/destroy_appliance_set.rb

```diff
@@ -0,0 +1,19 @@
+module Atmosphere
+  class DestroyApplianceSet
+    def initialize(appliance_set)
+      @appliance_set = appliance_set
+    end
+
+    def execute
```

Comment:
Can we use DB transaction on this entire execute operation? Otherwise we risk removing just a subset of Appliances from a given Set.